DOL.WHD.Section14c.Web/src/modules/pages/employerRegistrationTemplate.html

```diff
@@ -0,0 +1,122 @@
+<dol-header></dol-header>
+<main-top-nav-control></main-top-nav-control>
+<div ng-show="registrationSuccess">
+  <div  class="dol-feedback-success">
+    <p class="dol-feedback-message">
+      <strong>You Have Created an Employer</strong>
+    </p>
+  </div>
+  <p>Your account is now the point of contact for the employer you just registered. </p>
+  <p>If you have any quesitons contact us at 14c-help@dol.gov.</p>
+</div>
+<div ng-show="previouslyRegistered.status">
+  <div  class="dol-feedback-error">
+    <p class="dol-feedback-message">
+      <strong>We've run into a problem</strong>
+    </p>
+  </div>
+  <p><strong>This employer already has a registered account.</strong></p>
+  <p>Please contact {{previouslyRegistered.name}}, the point of contact for this employer, to request access to the account.</p>
+  <p>If you have any questions, you can reach us at 14c-help@dol.gov.</p>
+</div>
+<form ng-hide="registrationSuccess || previouslyRegistered" ng-submit="onSubmitClick()" class="dol-form-page">
+  <fieldset class="dol-form-sub-section">
+    <legend><h3>Register an Employer</h3></legend>
+    <hr />
+    Before you can begin the application, please register an employer for this account.
+    <div class="dol-form-question-block" ng-class="validate('employer.legalName') ? 'usa-input-error' : ''">
+      <div class="dol-form-question-help">
+        <label for="legalName" class="dol-form-question-text" id="legalNameLabel">Has this employer ever previously held a 14(c) certificate?</label>
```

Comment:
It looks to me like the `for`, `id`, and content of this `<label>` element all go with different bits of the form.  The attributes go with `legalName` and the text content asks whether the employer previous held a 14(c) certificate, but the section of the form is about the employer's trade name.

The way it's being used, I'm guessing this is really about whether the employer has previously held a 14(c) certificate, in which case the names and IDs should be changed.

Reply:
good catch. I'm realizing there are probably a bunch of these types of errors. Sorry!